api/lib/mno_enterprise/support_search.rb

```diff
@@ -43,6 +43,13 @@ def search
 
     private
 
+    def format_params(params)
+      {
+        org_search: ((params[:org_search] && JSON.parse(params[:org_search])) || { where: nil }).with_indifferent_access,
```

Comment:
Could you just have an empty hash instead of `where: nil` as you're using `dig`?

Reply:
ah yes -- agreed.